engine/version/old_records_cleaner.cpp

```diff
@@ -8,6 +8,29 @@
 #include "../sorted_collection/skiplist.hpp"
 
 namespace KVDK_NAMESPACE {
+
+void OldRecordsCleaner::DelayFree(void* addr, TimeStampType ts) {
```

Comment:
for alignment, use segnature PushToCache(PendingFreeSpaceEntry);

Reply:
That's Ok, but I don't suggest construct a struct and use it as parameter.
It's error prone, as you can easily forget a parameter or two.